docs/release.rst

```diff
@@ -310,6 +342,61 @@ Check if everything is correctly set:
    make check-branching
 
 
+Commit the changes:
+
+::
+
+   git commit -m "Adjust configuration for Fedora <version> (#infra)"
+
+
+Then merge this changes to the master branch:
+
+::
+
+   git checkout master
+   git merge f<version>-devel
+
+
+And revert the configuration changes on the master branch:
+
+::
+
+   git log   # to find SHA of the configuration commit
+   git revert <SHA>  # please add commit message why you are doing the revert
+
+The change revert has to be done if these branch specific configuration changes. Otherwise we won't
+be testing Rawhide on master.
+
+Adjust configuration on master
```

Comment:
For the other branches you explicitly mention `git checkout ...`, so master should have this too.